plugins/test/meson.build

```diff
@@ -5,6 +5,10 @@ if get_option('plugin_dummy')
   install_dummy = true
 endif
 
+install_data(['test-ble.quirk'],
+  install_dir: join_paths(datadir, 'fwupd', 'quirks.d')
+)
+
```

Comment:
Is there another way to key off knowing this is a ble device without having to quirk?  I would think there should be something we can look at in udev for example?

Reply:
Maybe all BLE devices should all be BLE/ rather than USB/ afterall?

Reply:
I think that would certainly make it a lot clearer yes.

Reply:
I went with BLUETOOTH as it's not BLE specific at all.